matterbridge/matterbridge-config.enc.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Secret
+stringData:
+    matterbridge.toml: ENC[AES256_GCM,data:bgMn/C9uGMQVBcpdN1qk5qcNdKuLgmKoK34ike5pO5GK8QRb2yIA7kDPtkmqMVE/4skKLwAyB/T4aKSFC3YlaV7p/WXlitr7jy43Gi7yCP5lEtXcvJFTXX6yFUdmhGXvNM8yaR2bs5Pgohz+iaBvP9Wm3vOM9YVrCZ0dwtQKMCv531gYfi0e9moL8kwbN342P+cKO6JpiytlU27kcrDmMU633myOajd9ZMW2PahyIcMqT60+fhc7gQm9bnBqcoteJ1b98/I8YaIWjl6VPS5QNHANJrcjPgloFr0dabBmYbPQT57vO7e1dREhB03MJY1AvZEgTavQ6qSHa3s3eFxXaijRjjbCIiqz7HLfmCtDeutLfCzivruhLtd+mOyH+v7TK7KTDbK/MuekY5QlnDBZQ+ga6zq3VUFYBc1DW0xnbbkmcqLRvywOLE7nhtuwJnyDUOGWVoZSvSXLnAvGiVMhxkCbPZvvfcWrOnCs9C8BNuVAiS9JEDTSjRUYVqilTTcoxlCLCvPXCf9lY7RHcrAm92vTaPb/F0Xf78DuXtrq+qKoWwLJizLgmd+3KEza3s8VWUk8xj1kAH9/w+XzIdhYI9euZIMKMABEeVyWJm9fjJNnbY1eAbU1oge56RDI9z/nQIT8955IE1ZpfH+1Bj1TcQTfs6NWbGpyYLNJFBXeMeLt6K03ytUFjWhB/xzgbJZqKsKV3trQx2ff8UdOrifyGczrLeC706ywIP0821BHIRRRj4rFyc+tTvULAa/N6W1jfPQNL/kQ4PSHN6Vjgs7NmDoBweNafJhWxy+N4bUvBWHmYjK2g1K455ySt78QUfb+gFjEkJTUejmATe91HeKyaneij12qQaH/79NwE0JLHY9g+iEZZaw9v5QQahw73/NC0KZBWQ5p97jA0Z1ba1QAe7ZuH9/h5/xKlcOb+ggOpronNZFxJ0+sUiMSOIz27kfPEvU59xX0mNV7IUwtsfyAu7kxhs3RwWrofs3h3jwYVZt8P/sJ6u0FKTGZ,iv:z7+f8tzarpL7KDFFLzNcb+pVHWdk8Xfm8QAypHIK1FM=,tag:tvoV97oQxzpEtIrc75WyNw==,type:str]
```

Comment:
is there any nice way to break this out?

Reply:
not without an envsubst init container; i figured this was simpler. if you'd like i can make one

Reply:
correction: possible, but poorly documented. will test.